builder/builder.go

```diff
@@ -643,6 +656,43 @@ func (b *Builder) AddBundles(bundles []string, alllocal bool, allupstream bool,
 	return nil
 }
 
+// ListBundles prints out the bundles in the mix, all bundles in the mix
+// including recursive includes, all bundles available in local bundles, or
+// all bundles available in upstream bundles
+func (b *Builder) ListBundles(full bool, local bool, upstream bool) error {
+	var bundles []string
+	var err error
+
+	switch {
+	case local:
+		bundles, err = b.getDirBundleList(b.Lbundledir)
+		if err != nil {
+			return errors.New("Error retreiving local bundle list")
```

Comment:
s/retreiving/retrieving/

Reply:
**sighs and repeats "i before e, except after c, ....**

Sorry.